newIDE/app/src/stories/componentStories/LayoutEditor/ObjectExporterDialog.stories.js

```diff
@@ -0,0 +1,43 @@
+// @flow
+
+import * as React from 'react';
+import { action } from '@storybook/addon-actions';
+
+// Keep first as it creates the `global.gd` object:
+import { testProject } from '../../GDevelopJsInitializerDecorator';
+
+import muiDecorator from '../../ThemeDecorator';
+import paperDecorator from '../../PaperDecorator';
+import ObjectExporterDialog from '../../../ObjectEditor/ObjectExporterDialog';
+import EventsFunctionsExtensionsContext from '../../../EventsFunctionsExtensionsLoader/EventsFunctionsExtensionsContext';
+import LocalEventsFunctionsExtensionWriter from '../../../EventsFunctionsExtensionsLoader/Storage/LocalEventsFunctionsExtensionWriter';
+import LocalEventsFunctionsExtensionOpener from '../../../EventsFunctionsExtensionsLoader/Storage/LocalEventsFunctionsExtensionOpener';
+
+export default {
+  title: 'LayoutEditor/ObjectExporterDialog',
+  component: ObjectExporterDialog,
+  decorators: [muiDecorator, paperDecorator],
+};
+
+const fakeEventsFunctionsExtensionsContext = {
+  loadProjectEventsFunctionsExtensions: async project => {},
+  unloadProjectEventsFunctionsExtensions: project => {},
+  unloadProjectEventsFunctionsExtension: (project, extensionName) => {},
+  reloadProjectEventsFunctionsExtensions: async project => {},
+  getEventsFunctionsExtensionWriter: () => LocalEventsFunctionsExtensionWriter,
```

Comment:
I wonder if this function should use the action method so that when you click on the button in the story, you can see the action has been called.

Reply:
I'm not sure because it's not really a user action, but rather internal to the implementation.